content/en/tracing/trace_collection/custom_instrumentation/go/migration.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I don't believe we are linking to this guide from anywhere else? If we want to drive users to upgrade, we could add a note by some of the other docs that mention v2 and point them here.